brave/brave-core
# Fixes twitch for the panel

Resolves https://github.com/brave/brave-browser/issues/5549

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:
- enable rewards
- go to twitch
- go to some channels and vods
- on every page open the panel and make sure that correct publisher is displayed in there

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
